lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -1604,21 +1626,21 @@ Value *PatchInOutImportExport::patchFsGenericInputImport(Type *inputTy, unsigned
 
       if (bitWidth == 16) {
         Value *args1[] = {
-            iCoord,                                               // i
-            ConstantInt::get(Type::getInt32Ty(*m_context), i),    // attr_chan
-            loc,                                                  // attr
-            ConstantInt::get(Type::getInt1Ty(*m_context), false), // high
-            primMask                                              // m0
+            iCoord,                                                  // i
+            ConstantInt::get(Type::getInt32Ty(*m_context), i),       // attr_chan
+            loc,                                                     // attr
+            ConstantInt::get(Type::getInt1Ty(*m_context), highHalf), // high
+            primMask                                                 // m0
```

Comment:
Please change the ConstantInt::get to use builder methods.

Reply:
Done